src/auth/auth.ts

```diff
@@ -766,7 +766,7 @@ export class Auth implements AuthService, ConnectionManager {
                 return partition(globals.context.globalState, devEnvId)
             }
 
-            return globals.context.workspaceState
+            return partition(globals.context.globalState, globals.hostname)
```

Comment:
Can we add a comment for context to future readers:

`// We are connected to a remote backend and want to maintain a separate state from our local state`